db/migrations/0004_alter_player_guild_alter_player_race.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-10-18 10:38
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_skill_race'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, to='db.guild'),
```

Comment:
The use of `on_delete=django.db.models.deletion.DO_NOTHING` might not be appropriate if the `player` model should not exist without the `guild` model. Consider using `on_delete=models.CASCADE` if the `player` should be deleted when the `guild` is deleted, or `on_delete=models.SET_NULL` if the `player` should remain with a null `guild` reference.